index.js

```diff
@@ -84,6 +84,10 @@ const camelCase = (input, options) => {
 		string => string.toLocaleUpperCase(options.locale);
 
 	if (input.length === 1) {
+		if (input.match(SEPARATORS)) {
```

Comment:
`RegExp#test` is the correct method to use here.

Reply:
done